backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -200,22 +200,24 @@ def _error_message(i: int, _url: str) -> dict:
 
 
 def verify_collection_body(body: dict, errors: list) -> None:
-    def check_if_blank(key) -> bool:
+    def check(key) -> bool:
         if key in body.keys():
             if not body[key]:
                 errors.append({"name": key, "reason": "Cannot be blank."})
+            elif re.search(r"[\x00-\x1f]", body[key]):
```

Comment:
I've seen some places include "\x7f" (DEL command) as a non-printable ascii character as well. Mind looking into it briefly to see if we should include in this regex? My instinct is yes

Reply:
I added a few more none printable characters to the regex